stock_picking_show_return/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================================
+Display returns on stock pickings
+=================================
+
+This module adds one tab in the pickings form view to display the returns
+pickings related to the current one.
+
+Usage
+=====
+
+#. Go to any picking that has a return, and see the new tab "Returns" appears.
```

Comment:
lol sorry but I think you misunderstood my comment.
I was talking that you do not write a list if there is only one item in it (English writing not python code) ;)